docusaurus/sidebars.js

```diff
@@ -1062,6 +1062,14 @@ const sidebars = {
               items: [
                 'dev-docs/migration/v4-to-v5/breaking-changes/removed-support-for-some-env-options',
                 'dev-docs/migration/v4-to-v5/breaking-changes/strict-requirements-config-files',
+                {
+                  type: 'doc',
+                  label: 'Server log level configuration',
```

Comment:
Same, please update the title. 

The goal is that by just looking at the title, users already get an idea of what has changed.
For instance: "Server log level configuration" → "Server log level is 'info'"